spec/generators/rspec/scaffold/scaffold_generator_spec.rb

```diff
@@ -208,16 +208,16 @@
       before { run_generator %w[posts upvotes:integer downvotes:integer] }
       subject { file("spec/views/posts/index.html.erb_spec.rb") }
       it { is_expected.to exist }
-      it { is_expected.to contain('assert_select "tr>td", text: 2.to_s, count: 2') }
-      it { is_expected.to contain('assert_select "tr>td", text: 3.to_s, count: 2') }
+      it { is_expected.to contain('assert_select cell_selector, text: Regexp.new(2.to_s), count: 2') }
+      it { is_expected.to contain('assert_select cell_selector, text: Regexp.new(3.to_s), count: 2') }
```

Comment:
How `cell_selector` is available in this context? Where is it defined?

Reply:
`cell_selector` is a variable defined in the scaffold tool.  Honestly, this spec is a little weird, as it basically just tests if the test generator generates the test it expects.  I'm not sure I see the point.